docs/import/manage_disaster.js

```diff
@@ -829,25 +853,37 @@ function createAssetDropdown(
     }
     select.append(assetOption);
   }
-  select.on(
-      'change', (event) => handleScoreAssetSelection(event, propertyPath));
 
   return select;
 }
 
 /**
- * Handles the user selecting an asset for one of the possible score types.
- * @param {Event} event Event, used to identify the target
+ * Adds the default change handler, which updates our internal data (and
+ * Firestore) when this element changes.
+ * @param {JQuery<HTMLElement>} elt
+ * @param {Array<string>} propertyPath The path to the value of this element
+ * @return {JQuery<HTMLElement>} The passed-in element, for chaining
+ */
+function addChangeHandler(elt, propertyPath) {
+  return elt.on(
+      'change',
+      (event) =>
+          handleScoreAssetSelection($(event.target).val(), propertyPath));
+}
+/**
+ * Handles the user entering a value into score-related input
+ * @param {string} val Value of input. 'None'/empty are treated like null (ugh)
```

Comment:
TODO to not have this? I think we can just have empty without None probably?

Reply:
You're right, got rid of it.